runtime/altair/src/lib.rs

```diff
@@ -329,6 +325,10 @@ impl pallet_balances::Config for Runtime {
 	type DustRemoval = ();
 	/// The minimum amount required to keep an account open.
 	type ExistentialDeposit = ExistentialDeposit;
+	type FreezeIdentifier = ();
+	type HoldIdentifier = ();
+	type MaxFreezes = ();
```

Comment:
**MAYBE CR**

What does this mean? Wording makes it sound like it should be an integer...

Reply:
Made it consistent with Polkadot/Moonbeam.